packages/modular-scripts/src/build.ts

```diff
@@ -559,8 +573,16 @@ export async function build(
   // ensure the root build folder is ready
   await fse.mkdirp(outputDirectory);
 
-  // delete any existing build folder, if at all
-  await prom(rimraf)(path.join(packagesRoot, directoryName, outputDirectory));
+  // delete any existing local build folders
```

Comment:
What do we do if one of these commands fails? People are left with half created modules? Maybe we should add some cleanup here so we don’t leave rouge files around.

Reply:
Also one of the followup items in the list at the bottom of this file.